samples/DevTestLabs/Modules/Library/Az.DevTestLabs2.psm1

```diff
@@ -713,6 +713,43 @@ function Set-AzDtlLabSharedImageGallery {
   }
 }
 
+function Get-AzDtlVirtualNetworks {
+  [CmdletBinding()]
+  param(
+    [parameter(Mandatory=$true, ValueFromPipeline=$true, HelpMessage="Lab object to get the Virtual Network from")]
+    [ValidateNotNullOrEmpty()]
+    $Lab,
+
+    [parameter(Mandatory=$false, ValueFromPipeline=$true, HelpMessage="If set, get the Expanded Resource from the std VNet object")]
+    $ExpandResource
+  )
+
+  begin {. BeginPreamble}
+  process {
+    try {
+      foreach($lab in $Lab) {
+
+        $lab = $lab | Get-AzDtlLab
+
+        $dtlLabVNets = Get-AzureRmResource `
+                          -ResourceGroupName $lab.ResourceGroupName `
+                          -ResourceType Microsoft.DevTestLab/labs/virtualnetworks `
+                          -ResourceName $lab.Name `
+                          -ApiVersion 2018-09-15 
+        
+        if ($ExpandResource) {
+          foreach($dtlLabVNet in $dtlLabVNets) {
+            Get-AzureRmVirtualNetwork -ResourceGroupName $dtlLabVNet.ResourceGroupName -Name $dtlLabVNet.Name -ExpandResource $ExpandResource
```

Comment:
If ExpandResource isn't passed in - does anything get returned from this commandlet?  I would think we should have the Lab VNets always returned, and if ExpandResource is passed in, we can inject another property that contains the underlying VNet resource?  There's an example for Getting VMs & Extended status in the library...